dm/relay/relay.go

```diff
@@ -581,16 +553,16 @@ func (r *Relay) handleEvents(
 	ctx context.Context,
 	reader2 Reader,
 	parser2 *parser.Parser,
-) (int, error) {
```

Comment:
please remove this comment  of this func `// the first return value is the index of last read rows event if the transaction is not finished.`